server/sonar-server/src/main/java/org/sonar/server/permission/PermissionPrivilegeChecker.java

```diff
@@ -55,10 +55,12 @@ public static void checkProjectAdmin(UserSession userSession, String organizatio
    * @deprecated does not support organizations. Replaced by {@link #checkProjectAdmin(UserSession, String, Optional)}
    */
   @Deprecated
-  public static void checkProjectAdmin(UserSession userSession, Optional<ProjectId> projectId) {
+  public static void checkProjectAdmin2(UserSession userSession, Optional<ProjectId> projectId) {
```

Comment:
the suffix `2` is a typo. Fixed. 
The method correctly handles organizations because `userSession.checkPermission(SYSTEM_ADMIN)` is replaced by `userSession.checkIsRoot()`.